app/components/global-error/global-error.tsx

```diff
@@ -27,8 +32,20 @@ export const GlobalErrorToast: ComponentType = () => {
   }
 
   if (networkError.statusCode >= 400 && networkError.statusCode < 500) {
-    // TODO translation
-    toastShow("Request issue.\nContact support if the problem persists")
+    const errorCode = (networkError as ServerError).result?.errors?.[0]?.code
+    switch (errorCode) {
+      case NetworkErrorCode.InvalidAuthentication:
+        toastShow(translate("common.logoutOnInvalidToken"), {
```

Comment:
I think the error message displayed should just say "You have been logged out.".  Saying the token has expired probably means nothing to most users and so best to keep the language as plain as possible.  We already have an error message we use when a user logs themselves out [here](https://github.com/GaloyMoney/galoy-mobile/blob/52efe8baf99d0831dac98cbb3a26d0dd70eb95a5/app/screens/authentication-screen/authentication-screen.tsx#L115).

Reply:
> Hi @rimbener. First of all thanks very much for your contribution. It's a really good first PR. I've made a couple of small comments. We would be happy to expand this to include your suggested TODO's as well if you are interested?

Sure!, I'll do the suggested changes and also will work on those TODO's.